src/definitions/modules/dropdown.js

```diff
@@ -345,11 +346,19 @@ $.fn.dropdown = function(parameters) {
             if( module.is.search() && !module.has.search() ) {
               module.verbose('Adding search input');
               $search = $('<input />')
-                .addClass(className.search)
-                .prop('autocomplete', 'off')
+                  .addClass(className.search)
+                  .prop({
+                      'autocomplete': 'off',
+                      'title': title,
+                      'disabled': module.is.disabled()
```

Comment:
Adding the `disabled` property to the search input at setup _once_ will break existing code which is possibly adding/removing the `disabled` class to the dropdown within the application at runtime without also removing/adding the disabled property for the search input.
I suggest to remove it here and add it as an enhancement in a separate PR, which will then be postponed for merge until 2.8.x. You should also provide a behavior `set disabled` and `remove disabled` which has to take care of the removement of the class as well as the removement of the property in that mentioned additional PR.

See breaking examples here http://jsfiddle.net/g5to4xks/

![dd_search_disabled_property](https://user-images.githubusercontent.com/18379884/58017287-1ab81980-7b00-11e9-8988-01021e12b879.gif)